Sources/A11yUITests/Tests/Values.swift

```diff
@@ -13,4 +13,5 @@ public struct A11yValues {
     public static let minMeaningfulLength = 2
     public static let maxMeaningfulLength = 40
     public static let allInteractiveElements = true
+    public static let floatComparisonTolerance: CGFloat = 0.001
```

Comment:
Thank you for this.
Do you think 0.001 is going to be reliable enough? When rendered I think 43.9 would meaningfully be equal to 44 so a threshold of 0.1 would be acceptable and we can ignore anything after the first decimal place. What are your thoughts?

Reply:
That makes sense to me. From an accessibility tap target perspective, even 0.1 would be unnoticeable in practice. I've revised as suggested.